books/free-programming-books-ar.md

```diff
@@ -38,6 +39,11 @@
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
 
 
+### Go
+
+* [تعلم لغة Go بسهولة](https://itwadi.com/learn-go-easily) - Alan Donovan, Brian Kernigan, مجموعة عمل (PDF)
+
+
```

Comment:
Patch to remove not authorized translation (see https://github.com/EbookFoundation/free-programming-books/pull/6497#discussion_r739666444)

```suggestion
```